### ISSUE TITLE
⛔ 🐞 Bug Report: SVG spacer image pushing down sticky header spacer

### ISSUE THREAD

**Issue by jamieburchell** [elementor/elementor#22476]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When adding an SVG to a sticky header, sometimes the content beneath the header is pushed down. This appears to be due to a spacer SVG image (shown in screencast) with a height of 150px, which is part of the visibility hidden sticky header spacer.

https://watch.screencastify.com/v/J3ybYUWHvkSlWnW1sHOK

### Steps to reproduce

1. Add a sticky header
2. Add SVG logo to header
3. Add content underneath header
4. Refresh the page

I suspect this is an issue with lazy loaded images, but I haven't the time to debug further at the moment. From what I can tell, the image in the spacer doesn't always get processed and replaced with the image in the data attribute.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.5.16
	PHP Version: 8.0.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://example.local/wp
	Home URL: https://example.local
	WP Multisite: No
	Max Upload Size: 40 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /news/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: jamie@example.local
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Converter for Media
		Version: 5.8.6
		Author: matt plugins

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	Pods - Custom Content Types and Fields
		Version: 2.9.13
		Author: Pods Framework Team

	Rank Math SEO
		Version: 1.0.114
		Author: Rank Math

	W3 Total Cache
		Version: 2.3.2
		Author: BoldGrid

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence


== Must-Use Plugins ==
	Bedrock Autoloader
		Version: 1.0.3
		Author: Roots

	Disable status tests
		Version: 1.0.0
		Author: Roots

	Register Theme Directory
		Version: 1.0.0
		Author: Roots


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	google_maps: Active


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
Log: showing 12 of 122023-05-15 12:11:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-15 12:11:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]
2023-05-15 12:12:08 [info] Update DB has been started [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-15 12:12:08 [info] elementor::elementor_updater Started 
2023-05-15 12:12:08 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-15 12:12:08 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-15 12:12:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.2',
)]
2023-05-15 12:12:10 [info] Update DB has been started [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]
2023-05-15 12:12:10 [info] elementor-pro::elementor_pro_updater Started 
2023-05-15 12:12:10 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-15 12:12:10 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-15 12:12:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.1',
)]

PHP: showing 13 of 13PHP: 2023-05-15 13:16:06 [warning X 51][/var/www/vhosts/example.local/releases/5/web/app/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 13:16:11 [warning X 1][/var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/import-export/runners/import/templates.php::103] Undefined array key "conditions" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 13:16:59 [warning X 8][/var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-15 13:28:33 [warning X 43][/var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/modules/dynamic-tags/pods/tags/pods-base.php::35] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 13:32:46 [warning X 1][/var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(69): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/vhosts/example.local/releases/5/web/app/plugins/elementor/data/base/endpoint.php(162): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> create_items()
',
)]
PHP: 2023-05-15 14:44:16 [warning X 21][/var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-15 14:44:56 [warning X 4][/var/www/vhosts/example.local/releases/6/web/app/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 14:45:08 [error X 1][/var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(14, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/vhosts/example.local/releases/6/web/app/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /var/www/vhosts/example.local/releases/6/web/wp/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /var/www/vhosts/example.local/releases/6/web/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /var/www/vhosts/example.local/releases/6/web/wp/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /var/www/vhosts/example.local/releases/6/web/wp/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 14:45:19 [warning X 2][/var/www/vhosts/example.local/releases/6/web/app/plugins/elementor/core/files/file-types/svg.php::234] simplexml_load_file(): I/O warning : failed to load external entity "https://example.local/app/uploads/2023/05/logo.svg" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-16 11:18:55 [warning X 2][/var/www/vhosts/example.local/releases/6/web/app/plugins/elementor/core/files/file-types/svg.php::234] simplexml_load_file(): I/O warning : failed to load external entity "https://example.local/app/uploads/2023/05/page-curl.svg" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-16 12:34:22 [warning X 21][/var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-17 09:20:43 [warning X 2][/var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::265] Attempt to read property "ID" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-17 09:33:23 [error X 1][/var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(14, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /var/www/vhosts/example.local/releases/7/web/app/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /var/www/vhosts/example.local/releases/7/web/wp/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /var/www/vhosts/example.local/releases/7/web/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /var/www/vhosts/example.local/releases/7/web/wp/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /var/www/vhosts/example.local/releases/7/web/wp/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-05-16 10:50:15 [error X 1][https://example.local/app/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:681852] this.model.isValidChild is not a function 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by nicholaszein** [comment]
Hello @jamieburchell,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- After testing with the Site Logo widget, Icon widget, and Image widget, using an SVG Image (in this case the Elementor Logo) we could not reproduce your issue. SVG images are not lazy-loaded by Elementor, and it seems that your issue is being caused either by the "Converter for Media" or by the "W3 Total Cache" plugins.

### ✅ What you can do to solve your issue:
- Isolate the problem by deactivating all plugins except Elementor, and by clearing and deactivating all levels of cache.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor, please feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
